test_main.py

```diff
@@ -37,3 +39,31 @@ def test_import():
     finally:
         os.environ.clear()
         os.environ.update(orig_env)
+
+def test_url_caching(empty_deployment):
+    publisher = kcidb.mq.URLListPublisher(
+        os.environ["PROJECT"],
+        os.environ["KCIDB_UPDATED_URLS_TOPIC"]
+    )
+    publisher.publish([
+        "https://github.com/kernelci/kcidb/blob/main/test_kcidb.py",
+        "https://github.com/kernelci/kcidb/blob/main/get-url-stats"
```

Comment:
Use the URLs of the "raw" files. We would get a smaller and faster response, and they would cost GitHub much less.
E.g. the first one should be [https://raw.githubusercontent.com/kernelci/kcidb/main/test_kcidb.py](https://raw.githubusercontent.com/kernelci/kcidb/main/test_kcidb.py)

Reply:
got it